src/index.js

```diff
@@ -1 +1,35 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import * as assets from './variables';
+import navigation from './navigation';
+import home from './home';
+import menu from './menu';
+import contact from './contact';
+
+const globalContainer = document.querySelector('#content');
+globalContainer.append(navigation(),
+  home(assets.heroImage),
+  menu(assets.dishes),
+  contact(assets.followUs));
+
+function openTab(e) {
+  const tabcontent = document.getElementsByClassName('tabcontent');
```

Comment:
**It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))**

* [ ] usage of arrow functions instead of the `function` keyword

Reply:
Issue solved :) thanks for your Review.